roles/apache/templates/etc/apache2/ports.conf

```diff
@@ -0,0 +1,4 @@
+{% for port in apache.listen %}
+Listen {{ port }}
+NameVirtualHost {{ port }}
+{% endfor %}
```

Comment:
I can get rid of this if need be, I plagiarized this play book and pretty much left the apache role untouched. I can get rid of this template completely, I left it in for flexibility if for some reason someone ever wanted to add a new port to listen on.